test/apps/pty/session.c

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
The session does not appear to be related to pty. I suggest adding a separate directory for processes and placing the test in `process/session.c`.